client/src/components/DropdownMenu/DropdownMenu.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { DropdownMenu } from "./DropdownMenu.tsx";
+
+export default {
+  title: "Dropdown/DropdownMenu",
+  component: DropdownMenu,
+};
+
+const Template = (args) => <DropdownMenu {...args} />;
+const dropdown_content = <div>Dropdown Content</div>;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  // text
+  dropdown_content,
+  button_description: "Button",
+  dropdown_trigger_text: "Trigger Text",
+  dropdown_a11y_text: "Accessibility Text",
```

Comment:
Ah, haha, had to stare at this/`DropdownMenu.tsx` for a while to figure out what was up, but these props are typoed. It's `..._txt` not `..._text` :shrug:

Kind of bugs me now, no benefit to dropping a single character, just setting up confusion like this. Maybe just update the actual component/consumers of the component to fully spell out text.

Reply:
Done! I ended up just following the component's spelling of txt, since there are other components that use DropdownMenu.